app/utils/file/index.ts

```diff
@@ -362,7 +362,7 @@ export function getLocalFilePathFromFile(serverUrl: string, file: FileInfo | Fil
                 }
             }
 
-            return `${FileSystem.CachesDirectoryPath}/${server}/${filename}-${hashCode(file.id!)}.${extension}`;
+            return `${FileSystem.CachesDirectoryPath}/${server}/${filename}-${file.id!}.${extension}`;
```

Comment:
I think we should apply here the fix for https://mattermost.atlassian.net/browse/MM-46320 unless I'm mistaken and is not this one?